### DISCUSSION TITLE
How can I earn a Galaxy Brain badge

### DISCUSSION THREAD

**Discussion by Likaius** [community/community#30782] [Other Feature Feedback, Questions, & Ideas]
I really wanna earn it

**Comment by snow-sr** [comment]
Just make an correct answer to some community post

**Comment by karthikeya-Git** [comment]
Hey @Likaius,

About that badge: -

Answer at least 2 discussions.
Take note, you CANNOT mark your own answers to your own questions. They have to be marked by another user.
Discussions should be in a public repository.
Also check this out, there is a whole conversation about that badge: -

(https://github.com/orgs/community/discussions/18293)

**Reply by karthikeya-Git** [reply]
And also, I forget to mention here is all achievement lists also check this for future reference
[GitHub-Achievement](https://github.com/Schweinepriester/github-profile-achievements)

**Comment by github-staff** [comment]
We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. 

Any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
